pages/common/zoxide.md

```diff
@@ -0,0 +1,29 @@
+# zoxide
+
+> Keep track of the most frequently used directories.
+> Uses a ranking algorithm to navigate to the best match.
```

Comment:
```suggestion
> Use a ranking algorithm to navigate to the best match.
```

I'm not sure weather should we use `s` here. I remember I read some guideline before. But now I cannot found out.
@zdroid @sbrl  What do you think here?

Reply:
My choice would be for `Uses a`. The alternative imperative sentence reads too much like a command to the user, rather than a description of the tool.

Reply:
`Uses`, since it's talking about what program itself does, not what you do with the program.

Reply:
It is okay then. I mark this as resolved.